src/routes/Categories.jsx

```diff
@@ -0,0 +1,10 @@
+export default function Categories() {
+  return (
+    <div>
+      <h1 className="go-down">This is Categories Page</h1>
+      <p>
+        For Now it is empty. Data will follow soon.
+      </p>
+    </div>
+  );
```

Comment:
- [ ] Sorry I am not sure you committed and pushed your changes. kindly render the button `check status` as the previous reviewer said.

Reply:
added
You can check now